SDPi_Supplement/asciidoc/volume2/dev-25/tf2-ch-a-mdpws-dev-25.adoc

```diff
@@ -0,0 +1,48 @@
+:var_uri_ws_discovery_transfer: https://www.w3.org/Submission/2006/SUBM-WS-Transfer-20060927/#Get
+:var_uri_dpws: http://docs.oasis-open.org/ws-dd/dpws/1.1/os/wsdd-dpws-1.1-spec-os.html#_Toc228672092
+[#vol2-clause-appendix-mdpws-dev-25]
+==== MDPWS: Discover BICEPS Services [DEV-25]
+
+===== GetMetadata Message
+
+The GetMetadata message is encoded by using {var_uri_ws_discovery_transfer}[WS-Get].
+
+====== Referenced Standards
+
+* <<ws_transfer_2006>> {var_uri_ws_discovery_transfer}[Section 3.1 Get]
+* <<dpws_2009>> {var_uri_dpws}[Section 4]
+
+====== Message Outline
+.GetMetadata message
+[source#vol2_clause_appendix_a_mdpws_dev_25_transfer_get,xml]
+----
+include::../../listings/vol2-clause-appendix-a-mdpws-dev-25-transfer-get.xml[]
+----
+
+====== Message Semantics
+The GetMetadata message does not contain any further semantics.
+
+===== GetMetadataResponse Message
+
+The GetMetadataResponse message is encoded by using {var_uri_ws_discovery_transfer}[WS-Get].
+
+====== Referenced Standards
+
+* <<ws-transfer-2006>> {var_uri_ws_discovery_transfer}[Section 3.1 Get]
+* <<dpws-2009>> {var_uri_dpws}[Section 4]
+
+====== Message Outline
+.GetMetadataResponse message
+[source#vol2_clause_appendix_a_mdpws_dev_25_transfer,xml]
+----
+include::../../listings/vol2-clause-appendix-a-mdpws-dev-25-transfer-response.xml[]
+----
+
+====== Message Semantics
+
+`s12:Envelope/s12:Body/wsm:Metadata/wsm:MetadataSection/dpws:ThisModel`:: The <<actor_somds_provider>>'s maps to the <<message_semantics_discover_biceps_services_model, Model Metadata>>.
+
+`s12:Envelope/s12:Body/wsm:Metadata/wsm:MetadataSection/dpws:ThisDevice`:: The <<actor_somds_provider>>'s maps to the <<message_semantics_discover_biceps_services_device, Device Metadata>>.
+
+`s12:Envelope/s12:Body/wsm:Metadata/wsm:MetadataSection/dpws:Relationship`:: The metadata indicating the relationship between a hosted service and its hosting service.

```

Comment:
The message semantics miss a mapping to `BICEPS Services`. This should be further specified, e.g. with a specific list similar to how I did it for DEV-27: https://github.com/IHE/sdpi-fhir/blob/84-append-dev-27-dgr/SDPi_Supplement/asciidoc/volume2/dev-27/tf2-ch-a-mdpws-dev-27.adoc. At least it needs to be mentioned somewhere. The actual list of allowed/necessary services can be provided later.

Reply:
```suggestion
`s12:Envelope/s12:Body/wsm:Metadata/wsm:MetadataSection/dpws:Relationship/dpws:Hosted/dpws:Types`:: Designates available <<payload_dev_25_get_metadata_biceps_services>> by providing a list of service types that contains at least the BICEPS GET SERVICE service type.
```